crates/uv/tests/it/python_install.rs

```diff
@@ -876,3 +876,43 @@ fn python_install_preview_broken_link() {
         );
     });
 }
+
+#[cfg(target_os = "macos")]
+#[test]
+fn python_dylib_install_name_is_patched_on_install() {
+    use assert_cmd::assert::OutputAssertExt;
+
+    let context: TestContext = TestContext::new_with_versions(&[]).with_filtered_python_keys();
+
+    // Install the latest version
+    context
+        .python_install()
+        .arg("--preview")
+        .arg("3.13")
+        .assert()
+        .success();
+
+    let dylib = context
+        .temp_dir
+        .child("managed")
+        .child("cpython-3.13.1-macos-aarch64-none")
```

Comment:
This bit is not going to fly, since it may run on multiple platform, not just "aarch64-none". 
I haven't found a way to get the "current" platform tag though. Any pointers? @zanieb

Reply:
Perhaps `platform_key_from_env`

Reply:
I did this in https://github.com/astral-sh/uv/pull/10629/commits/4f2dfbabe3118fd982d20da32a08e9844d2729e0 quick so we can release it today.

Reply:
Thank you!